tgb/linkproppred/negative_generator.py

```diff
@@ -245,11 +236,11 @@ def generate_negative_samples_hist_rnd(
         ], "Invalid split-mode! It should be `val` or `test`!"
 
         if os.path.exists(filename):
-            print(
+            vprint(
                 f"INFO: negative samples for '{split_mode}' evaluation are already generated!"
             )
         else:
-            print(f"INFO: Generating negative samples for '{split_mode}' evaluation!")
+            vprint(f"INFO: Generating negative samples for '{split_mode}' evaluation!")

```

Comment:
```suggestion
            vprint(f"Generating negative samples for '{split_mode}' evaluation!")
```
For consistency I would drop this pseudo logging style.

Reply:
Not essentially, leaving it as is for now